tests/suites/test_suite_aes.function

```diff
@@ -406,6 +408,20 @@ void aes_invalid_param( )
     TEST_INVALID_PARAM_RET( MBEDTLS_ERR_AES_BAD_INPUT_DATA,
                             mbedtls_aes_xts_setkey_dec( &xts_ctx, NULL, 128 ) );
 #endif
+
+
+    TEST_INVALID_PARAM_RET( MBEDTLS_ERR_AES_BAD_INPUT_DATA,
+                            mbedtls_aes_crypt_ecb( NULL,
+                                                   MBEDTLS_AES_ENCRYPT, in, out ) );
+    TEST_INVALID_PARAM_RET( MBEDTLS_ERR_AES_BAD_INPUT_DATA,
+                            mbedtls_aes_crypt_ecb( &aes_ctx,
+                                                   42, in, out ) );
```

Comment:
Minor: As suggest by @k-stachowiak in another PR, use named constants for valid and invalid modes. I.e. `int valid_mode = MBEDTLS_AES_ENCRYPT`, `int invalid_mode = 42`. Not a blocker.